.changeset/moody-nails-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": major
```

Comment:
```suggestion
"@hashicorp/design-system-components": patch
```
Was the use of `major` here intentional?

Reply:
Oh, no, certainly not intentional. Fixed now! 🙏